cucumber/authenticators_jwt/features/authn_status_jwt.feature

```diff
@@ -5,7 +5,8 @@ Feature: JWT Authenticator - Status Check
 
   @sanity
   Scenario: ONYX-9122: A valid JWT status request, 200 OK
-    Given I load a policy:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
```

Comment:
Use remote endpoint like the other tests

Reply:
Why?